### PR TITLE
GC blog post: latency rematch

### PR DESCRIPTION
**Pull request by mtopolnik**

### CODE REVIEW

File: site/website/blog/2020-06-25-jdk-gc-benchmarks-rematch.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+be released on July 14. To nevertheless make progress, we took the
+source code from the jdk14u tree, at the changeset number
+[57869:e9d41bbaea38](http://hg.openjdk.java.net/jdk-updates/jdk14u/rev/e9d41bbaea38),
+and applied the changeset number
+[59746:29b4bb22b5e2](https://hg.openjdk.java.net/jdk/jdk/rev/29b4bb22b5e2)
+from the main jdk tree on top of it. The jdk14u tree is where JDK 14.0.2
+will be released from and the changeset 59746:29b4bb22b5e2 applies the
+patch resolving the mentioned Shenandoah issue.
+
+`-XX:-UseBiasedLocking`
```
Comment (jbartok) [inline site/website/blog/2020-06-25-jdk-gc-benchmarks-rematch.md:50]: Maybe say something about these config options, they just feel orphan without any explanation.

**Comment by viliam-durina** [comment]
We could maybe crosslink the series articles: add links to other articles of the series.

**Comment by mtopolnik** [comment]
> We could maybe crosslink the series articles: add links to other articles of the series.

There's already a link to the 1st post right at the start of the article (the word "article" is a hyperlink to it).

**Comment by cangencer** [comment]
> > We could maybe crosslink the series articles: add links to other articles of the series.
> 
> There's already a link to the 1st post right at the start of the article (the word "article" is a hyperlink to it). You mean add a link to the new article from the previous ones?

I think he meant adding the link to the old articles, I agree there should be a link to the update from part1.

**Comment by viliam-durina** [comment]
I meant to add something like this into each article in the series:
>This article is a part of series:
>1. First part link
>2. Second part link (you're viewing this)
>3. Third part link

Or at least a link from the old articles to this new one.

**Pull request merged**